challenges/recruitment.md

```diff
@@ -0,0 +1,21 @@
+# Recruitment
+
+## Job Application Process
+
+To avoid bias in the recruitment process you can use [Gender Decoder](https://gender-decoder.katmatfield.com/)
+
+## Interview Process
+
+### Technical Interview
+
+Some institutions use a technical challenge as part of the interview process
+This is sometimes used to also test soft skills such as "how do you approach a
+problem" and "how do you communicate your solution".
+
+## Job description
+
+- We can split job adverts into two sections.
+- Expectation of the role. This should be very general to ensure that the RSE
+  can adapt to a variety of projects and roles.
+- Skills and competencies. This should be more specific and can be tailored to
```

Comment:
I would say all parts of the JD should be general and kept consistent across hiring rounds. This helps if you're looking to implement a promotion scheme or similar. It's the job **advert** where you can be specific with extra requirements for a particular hire.